StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/USBankAccount/BankAccountInfoView.swift

```diff
@@ -82,31 +119,23 @@ class BankAccountInfoView: UIView {
 
     func addViewComponents() {
         bankIconImageView.translatesAutoresizingMaskIntoConstraints = false
-        bankNameLabel.translatesAutoresizingMaskIntoConstraints = false
-        bankAccountNumberLabel.translatesAutoresizingMaskIntoConstraints = false
         xIcon.translatesAutoresizingMaskIntoConstraints = false
         xIconTappableArea.translatesAutoresizingMaskIntoConstraints = false
 
         addSubview(bankIconImageView)
-        addSubview(bankNameLabel)
-        addSubview(bankAccountNumberLabel)
+        addSubview(contentStackView)
         xIconTappableArea.addSubview(xIcon)
         addSubview(xIconTappableArea)
 
         NSLayoutConstraint.activate([
             bankIconImageView.centerYAnchor.constraint(equalTo: centerYAnchor),
             bankIconImageView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.spacing),
-
-            bankNameLabel.leadingAnchor.constraint(equalTo: bankIconImageView.trailingAnchor, constant: Constants.spacing),
-            bankNameLabel.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor, multiplier: 0.5),
-            bankNameLabel.topAnchor.constraint(equalTo: topAnchor, constant: Constants.spacing),
-            bankNameLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -Constants.spacing),
-
-            bankAccountNumberLabel.leadingAnchor.constraint(equalTo: bankNameLabel.trailingAnchor, constant: Constants.spacing),
-            bankAccountNumberLabel.topAnchor.constraint(equalTo: topAnchor, constant: Constants.spacing),
-            bankAccountNumberLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -Constants.spacing),
-
-            xIconTappableArea.leadingAnchor.constraint(greaterThanOrEqualTo: bankAccountNumberLabel.trailingAnchor, constant: Constants.spacing),
+            
+            contentStackView.leadingAnchor.constraint(equalTo: bankIconImageView.trailingAnchor, constant: Constants.spacing),
+            contentStackView.topAnchor.constraint(equalTo: topAnchor, constant: Constants.spacing),
+            contentStackView.trailingAnchor.constraint(lessThanOrEqualTo: xIconTappableArea.leadingAnchor),
```

Comment:
Just for my own understanding; this is saying that the `contentStackView` can horizontally expand all the way to the `xIconTappableArea.leadingAnchor`, but it doesn't take up more horizontal space than it needs?

Reply:
Yep, that’s the intention.